### PR TITLE
added categories page #6433

### PR DESCRIPTION
**Pull request by gautampanday**

### CODE REVIEW

File: theme/templates/index.html [hunk 1]
```diff
@@ -1,13 +1,26 @@
...
+      {% block breadcrumb %}
+         <li class="white-text-color">{% trans %}Blog{% endtrans %}</li>
+       {% endblock breadcrumb %}
     </ol>
+    <div class="text-center section-space">
+      {% block heading %}
+      <h1 class="white-text-color heading">{% trans %}Blog{% endtrans %}
+      </h1>
+      {% endblock heading %}
+      <div class="col-sm-12 col-xs-12" >
```
Comment (GauravButola) [inline theme/templates/index.html]: Empty div

**Comment by GauravButola** [comment]
LGTM

File: theme/templates/index.html [hunk 2]
```diff
@@ -1,13 +1,27 @@
...
+        <div class="col-sm-2 col-xs-4 col-xs-offset-4 col-sm-offset-5 app-banner-divider">
+          <!-- This is used for divider at banner -->
+        </div>
+      </div>
+    </div>
   </div>
 </section>
 <section class="section-space">
```
Comment (tarunbhardwaj) [inline theme/templates/index.html:29]: create content block here and override in further pages

File: theme/templates/categories.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
+{% set SITE_ROOT = '../' %}
+{% extends 'index.html' %}
+{% block main %}
+<section class="app-banner">
+  <div class="container">
+    <ol class="breadcrumb remove-background">
+      <li>
+        <a class="white-text-color" href="{{ SITE_ROOT }}index.html">{% trans %}Home{% endtrans %}</a>
+      </li>
+      {% block breadcrumb %}
```
Comment (tarunbhardwaj) [inline theme/templates/categories.html]: just override breadcrumb and heading block

**Comment by GauravButola** [comment]
LGTM

File: theme/templates/categories.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+            {% for category, articles in categories %}
+            <div class="panel panel-default">
+              <div class="panel-heading">
+                <h4 class="panel-title">
+                  <a data-toggle="collapse" data-parent="#accordion" href="#collapse-{{category.slug}}">{{ category }} <span class="badge pull-right">{{ articles|count }}</span></a>
+                </h4>
+              </div>
+              <div id="collapse-{{category.slug}}" class="panel-collapse collapse">
+                <div class="panel-body">
+                  {% for article in articles %}
```
Comment (tarunbhardwaj) [inline theme/templates/categories.html]: render all articles ?

**Comment by tarunbhardwaj** [comment]
Tag page is broken.  in tag page

``` jinja
{% set SITE_ROOT = '../../' %}
```

**Pull request merged**